### ISSUE TITLE
Class 'DartObjectImpl' has no instance getter 'value' => in 1.14.0-dev.5.0

### ISSUE THREAD

**Issue by Andersmholmgren** [google/built_value.dart#15]
I copied the example including the build file. When I run the build.dart it creates a value.g.dart but it is empty except for the error

Dart VM version: 1.14.0-dev.5.0 (Fri Dec 18 02:14:17 2015) on "macos_x64"

The stack trace was

```
null
  LibraryGenerationResultKind.noop
lib/src/value.g.dart
  LibraryGenerationResultKind.created
    Class 'DartObjectImpl' has no instance getter 'value'.
    
    NoSuchMethodError: method not found: 'value'
    Receiver: Instance of 'DartObjectImpl'
    Arguments: []
    #0      Object._noSuchMethod (dart:core-patch/object_patch.dart:42)
    #1      Object.noSuchMethod (dart:core-patch/object_patch.dart:45)
    #2      BuiltValueGenerator.generateCode.<anonymous closure>.<anonymous closure> (package:built_value_generator/built_value_generator.dart:210:55)
    #3      Object&ListMixin.any (dart:collection/list.dart:115)
    #4      BuiltValueGenerator.generateCode.<anonymous closure> (package:built_value_generator/built_value_generator.dart:209:74)
    #5      WhereIterator.moveNext (dart:_internal/iterable.dart:436)
    #6      MappedIterator.moveNext (dart:_internal/iterable.dart:389)
    #7      Iterable.contains (dart:core/iterable.dart:206)
    #8      BuiltValueGenerator.generateCode (package:built_value_generator/built_value_generator.dart:233:31)
    #9      BuiltValueGenerator.generate.<generate_async_body> (package:built_value_generator/built_value_generator.dart:53:12)
    #10     Future.Future.microtask.<anonymous closure> (dart:async/future.dart:144)
    #11     _microtaskLoop (dart:async/schedule_microtask.dart:41)
    #12     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50)
    #13     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
    #14     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:149)

```

**Comment by davidmorgan** [comment]
Thanks.

This looks like a change in the analyzer library -- I just ran "pub upgrade" and it started failing for me, too.

I'll look at a fix.

**Comment by davidmorgan** [comment]
Fixed in 0.0.5, please let me know if you see further issues. Thanks again!
